platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/utils/FontUtils.java

```diff
@@ -0,0 +1,61 @@
+package com.mapbox.mapboxsdk.utils;
+
+import android.graphics.Typeface;
+
+import com.mapbox.mapboxsdk.log.Logger;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Utility class to select a font from a range of font names based on the availability of fonts on the device.
+ */
+public class FontUtils {
+
+  private static final String TAG = "Mbgl-FontUtils";
+  private static final String DEFAULT_FONT = "sans-serif";
```

Comment:
I think it'd be great to move this into a public constant and reuse in the map options, snapshotter and tests.